packages/web-components/scripts/utilities.js

```diff
@@ -0,0 +1,22 @@
+import fs from 'fs';
+import { fileURLToPath } from 'url';
+import path from 'path';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.resolve(path.dirname(__filename), '../');
```

Comment:
this is an antipattern, ESM resolved `__dirname` should folllow CJS `__dirname`

it would be nice to refactor this utility to following:

```js
import fs from 'node:fs';
import { fileURLToPath } from 'node:url';
import path from 'node:path';

const __filename = fileURLToPath(import.meta.url);
export const __dirname = path.dirname(__filename);

function isComponentDirectory(item) {
  const directoryItems = fs.readdirSync(path.resolve(item.path, item.name), { withFileTypes: true });
  return directoryItems
    .map(directoryItem => {
      return directoryItem.name;
    })
    .includes('define.ts');
}

export function listComponentDirectories() {
  return fs
    .readdirSync(path.resolve(__dirname, '../src'), { withFileTypes: true })
    .filter(item => item.isDirectory() && isComponentDirectory(item))
    .map(directory => directory.name);
}

```